packages/orbit-components/src/Icon/IconList.tsx

```diff
@@ -1,62 +1,36 @@
 import * as React from "react";
-import styled from "styled-components";
-import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
 import * as Icons from "../icons";
 
-const List = styled.div`
-  display: flex;
-  flex-flow: row wrap;
-  justify-content: space-between;
-  border-radius: ${defaultTokens.borderRadiusLarge};
-`;
-
-const Container = styled.div`
-  display: flex;
-  flex-direction: row;
-  gap: ${defaultTokens.spaceLarge};
-  justify-content: flex-start;
-  align-items: center;
-  align-content: center;
-  width: 100%;
-  min-height: 80px;
-  background-color: white;
-  margin-bottom: ${defaultTokens.spaceLarge};
-  border-radius: ${defaultTokens.borderRadiusLarge};
-  border: ${defaultTokens.borderWidthCard} ${defaultTokens.borderStyleCard}
-    ${defaultTokens.paletteCloudNormal};
-  padding: 0 ${defaultTokens.spaceLarge};
-`;
-
-const IconImport = styled.div`
-  font-family: "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
-  font-size: 12px;
-  line-height: ${defaultTokens.lineHeightTextSmall};
-  color: ${defaultTokens.paletteInkDark};
-  background-color: ${defaultTokens.paletteCloudLight};
-  border: ${defaultTokens.borderWidthCard} ${defaultTokens.borderStyleCard}
-    ${defaultTokens.paletteCloudNormal};
-  padding: ${defaultTokens.spaceXXSmall} ${defaultTokens.spaceXSmall};
-`;
-
 const IconList = () => (
-  <List>
+  <div className="rounded-large flex flex-row flex-wrap justify-between">
     {Object.keys(Icons)
       .filter(n => n !== "__namedExportsOrder")
       .map(icon => {
         const Icon = Icons[icon];
 
         const iconName = `${icon}`;
         return (
-          <Container key={icon}>
+          <div
+            className="gap-lg bg-white-normal mb-lg rounded-large border-cloud-normal px-lg flex w-full flex-row content-center items-center justify-start border border-solid"
+            style={{ minHeight: "80px" }}
```

Comment:
Why not `min-h-[80px]`?

Reply:
I wanted to avoid that custom class to be built in the final CSS file. Since this component is to be used only in Storybook, I prefer the style to be inline, so that class is not unnecessarily created by the Tailwind compiler